### ISSUE TITLE
Resuming work - becoming more urgent

### ISSUE THREAD

**Issue by artydont** [ScientificPublishing/SciPub#16]
@ScientificPublishing/scipub I have created this team, please accept invitations to join it. It is necessary to @mention it on any new issue as other methods do not seem to result in everybody getting emails about new issues.

GitHub is now requiring all users to implement tighter security. I should have done so myself long ago but I have not and I will now do so as soon as I can. 

Please read [how and why to do it](https://github.blog/2023-03-09-raising-the-bar-for-software-security-github-2fa-begins-march-13/) and take the time to do it. It is both useful and necessary and now urgent.
 
I'm still focussed elsewhere but this project is becoming more urgent in view of the potential rapid change in environment for censorship and mass surveillance. In particular access for people with visual impairments will become more important.

1. [archive.today](https://en.wikipedia.org/wiki/Archive.today) and most of its domains has been down for several days. I was able to access it only via Brave browser Tor private page with long delay for onion address provided at above Wikipedia page:

[archive.onion](http://archiveiya74codqgiixo33q62qlrqtkgmcitqx5u2oeqnmn5bpcbiyd.onion/)

The onion link at bottom of the wikipedia box with list of domain URLs also has link [explaining Tor and onion access](https://en.wikipedia.org/wiki/Tor_(network)#Tor_Browser). Simplest approach, also for other related stuff is to use [Brave browser](https://brave.com/). The illusion of anonymity and privacy offered by onion is financed by US government and supervised by NSA. Effective only against surveillance by employers, not against governments. Willingness to put up with such hassles helps government intelligence agencies identify smaller numbers of potential targets of interest for more thorough surveillance than "mass surveillance".

2. This quite significant obstacle may be related to [court orders against Internet Archive](https://blog.archive.org/2023/08/17/what-the-hachette-v-internet-archive-decision-means-for-our-library/) and may point to things rapidly becoming more difficult.

I was able to use the onion address for archive today to save [this archived page](http://archiveiya74codqgiixo33q62qlrqtkgmcitqx5u2oeqnmn5bpcbiyd.onion/5pACj) about current events, (with major delay to pass verification checks. But links to it won't be easily accessed until there is widespread awareness and pages won't get saved without systematic organization of that widespread awareness. It is easy to imagine access to such information becoming confined to the margins.

3. Spending time to become more familiar with the technology could become high priority soon. The earlier you start the easier it will be.

**Comment by Tomism** [comment]
yes

**Comment by Ted1307** [comment]
I used archive.is on Wednesday 13 Dec. without any problem; however when I tried today (15 Dec) to look at the archived page you'd saved (point 2) I got the message that it has been blocked by Brave.

**Comment by artydont** [comment]
archive.is (and other domains) have been down for everyone for more than a week:

https://www.isitdownrightnow.com/archive.is.html

Perhaps cached pages were still available sometimes.

The link I provided that is blocked by Brave works only with Tor on a private tab and also requires repeatedly authenticating as human by identifying pictures of buses, traffic lights and fire hydrants. The service still exists but is currently effectively useless so I won't explain further how to access onion addresses via Brave browser using Tor (Brave does explain). 

Google for [archive.today down](https://www.google.com/search?q=archive.today%20down) will find other confirmation.

BTW I have only had a quick look at what is required by GitHub for authentication. Will write notes later suggesting avoidance of providing phone numbers and using SMS or apps installed on phones. Instead use a TOTP app on PC:

https://en.wikipedia.org/wiki/Comparison_of_OTP_applications

Will recommend specific app for each platform (Linux, Windows and Mac). Hopefully to also support authentication management generally and use with all forms of git repos and other secured sites while avoiding provision of identifying information such as use of a specific identifiable device such as a phone or phone number while enabling recovery of lost passwords etc.

My strong impression is that will become increasingly difficult and people who don't provide biometric identification using their phone will be shut out to the extent that those refusing can be more easily placed under special surveillance.

Long term something like [selfhosted Vaultwarden](https://github.com/dani-garcia/vaultwarden) could be installed automatically on all platforms but I doubt that is feasible at the moment. So will take some time for a recommendation.

**Comment by Ted1307** [comment]
Is it also a good idea to be using a VPN? In which case, are any better than others?

**Comment by artydont** [comment]
I'm not using VPN. Some sites blocked by Australian ISPs but I assume similar blocks elsewhere.

Access to alternative DNS root may be more useful but I have not looked into it and won't for a while:

https://en.wikipedia.org/wiki/Alternative_DNS_root


Becoming familiar with IPFS and related technologies will be most useful and part of this project but I don't have time at the moment:

https://en.wikipedia.org/wiki/InterPlanetary_File_System

**Comment by Tomism** [comment]
yes
